api/api.go

```diff
@@ -330,6 +330,46 @@ func postContainersExec(c *context, w http.ResponseWriter, r *http.Request) {
 	w.Write(data)
 }
 
+// DELETE /images/{name:.*}
+func deleteImages(c *context, w http.ResponseWriter, r *http.Request) {
+	if err := r.ParseForm(); err != nil {
+		httpError(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	var name = mux.Vars(r)["name"]
+
+	matchedImages := []*cluster.Image{}
+	for _, image := range c.cluster.Images() {
+		if image.Match(name) {
+			matchedImages = append(matchedImages, image)
+		}
+	}
+
+	if len(matchedImages) == 0 {
+		httpError(w, fmt.Sprintf("No such image %s", name), http.StatusNotFound)
+		return
+	}
+
+	out := []*dockerclient.ImageDelete{}
+	errs := []string{}
+	for _, image := range matchedImages {
+		content, err := image.Node.DockerClient().RemoveImage(name)
```

Comment:
@aluzzardi What about `cluster.RemoveImage(image)` right now the node has no actions, only "getters" and I think it's a good thing